tests/phpunit/integration/Core/Tags/Guards/Tag_Environment_Type_GuardTest.php

```diff
@@ -42,10 +42,36 @@ public function test_can_not_activate_in_development() {
 			$this->markTestSkipped( 'The uopz extension is not available.' );
 		}
 		$env_type      = wp_get_environment_type();
-		$tagproduction = new Tag_Production_Guard();
+		$tagproduction = new Tag_Environment_Type_Guard();
 		uopz_set_static( 'wp_get_environment_type', array( 'current_env' => 'development' ) );
 		$this->assertFalse( $tagproduction->can_activate() );
 		uopz_set_static( 'wp_get_environment_type', array( 'current_env' => $env_type ) );
 
 	}
+
+	public function test_can_activate_in_development() {
+		// Pre WP-5.5.0
+		if ( ! function_exists( 'wp_get_environment_type' ) ) {
+			$this->markTestSkipped( 'Missing wp_get_environment_type() function.' );
+		}
+		if ( ! function_exists( 'uopz_set_static' ) ) {
+			$this->markTestSkipped( 'The uopz extension is not available.' );
+		}
+		$env_type      = wp_get_environment_type();
+		$tagproduction = new Tag_Environment_Type_Guard();
+		uopz_set_static( 'wp_get_environment_type', array( 'current_env' => 'development' ) );
+		$this->assertFalse( $tagproduction->can_activate() );
+
+		add_filter(
+			'googlesitekit_allowed_tag_environment_types',
+			function ( $allowed_environments ) {
+				$allowed_environments[] = 'development';
+				return $allowed_environments;
+			}
+		);
```

Comment:
Let's make sure no other hooks are added to the `googlesitekit_allowed_tag_environment_types` filter before adding this one. Please, call the `remove_all_filters( 'googlesitekit_allowed_tag_environment_types' )` function before adding a new hook.